src/components/ElementResizeListener.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {
+  useCallback, useRef, useEffect, RefObject,
+} from 'react';
+
+interface Props {
+  onResize: (event: Event) => void;
+}
+
+const ElementResizeListener: React.FC<Props> = ({ onResize }) => {
+  const rafRef = useRef(0);
+  const objectRef: RefObject<HTMLObjectElement> = useRef(null);
+  const onResizeRef = useRef(onResize);
+
+  onResizeRef.current = onResize;
+
+  const localOnResize = useCallback((e: Event) => {
+    if (rafRef.current) {
+      cancelAnimationFrame(rafRef.current);
+    }
+    rafRef.current = requestAnimationFrame(() => {
+      onResizeRef.current(e);
+    });
+  }, []);
+
+  const onLoad = useCallback(() => {
+    const obj = objectRef.current;
+    if (obj && obj.contentDocument && obj.contentDocument.defaultView) {
+      obj.contentDocument.defaultView.addEventListener('resize', localOnResize);
+    }
+  }, [localOnResize]);
+
+  useEffect(() => () => {
+    const obj = objectRef.current;
+    if (obj && obj.contentDocument && obj.contentDocument.defaultView) {
+      obj.contentDocument.defaultView.removeEventListener('resize', localOnResize);
+    }
+  }, [localOnResize]);
+
+  return (
+    <object
```

Comment:
Why `<object>`? (not very familiar with this one 😄)

Reply:
The benefit of <object> is that you can add a 'resize' event listener to it. (I borrowed the ElementResizeListener.tsx from another project, but it seems to work well.)

Reply:
I believe you can add onResize to any HTML element (incl. e.g. `<div>`) but it doesn't matter, only curious.